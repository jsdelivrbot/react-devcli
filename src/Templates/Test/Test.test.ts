import {js, ts} from './Test';

describe('Test template test suite', (): void => {
    test('should match test template snapshot for JS language', (): void => {
        expect(js()).toMatchSnapshot();
    });

    test('should match test template snapshot for TS language', (): void => {
        expect(ts()).toMatchSnapshot();
    });
});
