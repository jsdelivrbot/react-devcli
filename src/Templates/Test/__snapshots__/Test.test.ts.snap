// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test template test suite should match test template snapshot for JS language 1`] = `""`;

exports[`Test template test suite should match test template snapshot for TS language 1`] = `""`;
